pkg/manager/member/node.go

```diff
@@ -51,3 +55,42 @@ func getNodeLabels(nodeLister corelisterv1.NodeLister, nodeName string, storeLab
 	}
 	return labels, nil
 }
+
+// IsNodeReadyConditionFalseOrUnknown returns true if a pod is not ready; false otherwise.
+func IsNodeReadyConditionFalseOrUnknown(status corev1.NodeStatus) bool {
+	condition := getNodeReadyCondition(status)
+	return condition != nil && (condition.Status == corev1.ConditionFalse || condition.Status == corev1.ConditionUnknown)
+}
+
+// IsNodeRODiskFoundConditionTrue returns true if a pod has RODiskFound condition set to True; false otherwise.
+func IsNodeRODiskFoundConditionTrue(status corev1.NodeStatus) bool {
+	_, condition := getNodeCondition(&status, nodeCondRODiskFound)
+	return condition != nil && condition.Status == corev1.ConditionTrue
+}
+
+// GetNodeReadyCondition extracts the node ready condition from the given status and returns that.
+// Returns nil if the condition is not present.
+func getNodeReadyCondition(status corev1.NodeStatus) *corev1.NodeCondition {
+	_, condition := getNodeCondition(&status, corev1.NodeReady)
+	return condition
+}
+
+// GetNodeCondition extracts the provided condition from the given status and returns that.
+// Returns nil and -1 if the condition is not present, and the index of the located condition.
+func getNodeCondition(status *corev1.NodeStatus, conditionType corev1.NodeConditionType) (int, *corev1.NodeCondition) {
```

Comment:
Similar comments here. Maybe we don't need to return the index since it's never used in this change set.
nit: make function names consistent with the ones in the comments.

Reply:
Made the changes to only return *corev1.PodCondition